models/issues/issue_project.go

```diff
@@ -51,7 +51,7 @@ func (issue *Issue) ProjectBoardID(ctx context.Context) int64 {
 func LoadIssuesFromBoard(ctx context.Context, b *project_model.Board) (IssueList, error) {
 	issueList := make(IssueList, 0, 10)
 
-	if b.ID != 0 {
+	if b.ID > 0 {
```

Comment:
I mean, it does make a difference if it were `db.NoConditionID`.

Reply:
No, we only use `db.NoConditionID` when searching the issue, in this case, it's the function `Issues`.  Actually, the `ProjectBoardID` can't be 0.